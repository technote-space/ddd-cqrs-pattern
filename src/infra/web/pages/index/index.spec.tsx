import renderer from 'react-test-renderer';
import { IndexPage } from '.';

describe('IndexPage', () => {
  it('should return IndexPage component', () => {
    const Component = (new IndexPage()).create();
    expect(Component.displayName).toBe('IndexPage');

    const tree = renderer.create(<Component/>);
    expect(tree).toMatchSnapshot();
  });
});
