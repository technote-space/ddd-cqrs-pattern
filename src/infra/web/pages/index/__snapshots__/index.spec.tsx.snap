// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage should return IndexPage component 1`] = `
<div>
  Hello World!
</div>
`;
